README.md

```diff
@@ -1,12 +1,76 @@
 # hello-world-fwless
-Simple java API rest without framework
+Simple java rest API without framework.
 
+This branch aims to showcase how to use [Dekorate](https://github.com/dekorateio/dekorate) to deploy this microservice on a Kubernetes cluster.
```

Comment:
... to generate the YAML resources and next deploy this microservice on a kubernetes cluster.